src/Css/Foreign.elm

```diff
@@ -364,6 +365,12 @@ withClass class =
     Preprocess.ExtendSelector (Structure.ClassSelector (identifierToString "" class))
 
 
+{-| -}
```

Comment:
@lukewestby mind adding an example so folks can see how to use this?

Reply:
@rtfeldman can i make a separate pull request that adds examples to all the combinators?

Reply:
Absolutely! That would be 😻

Reply:
To make the second PR easier, I'll merge this but hold off publishing until after that PR lands!